### ISSUE TITLE
conda install & running produces error

### ISSUE THREAD

**Issue by brookehus** [markovmodel/pyemma_tutorials#110]
I ran this, in an environment I had already created:

`conda install -c conda-forge pyemma_tutorials`

Then I ran:

`pyemma_tutorials`

And I got:

```
Traceback (most recent call last):
  File "/Users/brooke/miniconda3/envs/pyemma/bin/pyemma_tutorials", line 6, in <module>
    from pyemma_tutorials import main
ImportError: cannot import name 'main'
```

**Comment by marscher** [comment]
I think this is fixed on master. You can do python setup.py develop on
your git clone to make it work. I will issue a new conda-forge beta
version soon. Thanks for testing!

**Comment by brookehus** [comment]
The notebooks are working fine because I already have pyemma installed.

If I compile from source using `python setup.py develop` (or `install`) I get a different error:

```
(pyemma) [brooke@xenon:~/source/pyemma_tutorials (master)]$ pyemma_tutorials
Traceback (most recent call last):
  File "/Users/brooke/miniconda3/envs/pyemma/bin/pyemma_tutorials", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3098, in <module>
    @_call_aside
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3082, in _call_aside
    f(*args, **kwargs)
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3111, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 573, in _build_master
    ws.require(__requires__)
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/brooke/miniconda3/envs/pyemma/lib/python3.6/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'monotonic' distribution was not found and is required by humanfriendly
```

**Comment by marscher** [comment]
humanfriendly is a dependency of mdshare. @cwehmeyer did you yet encounter any problems with installing mdshare completely with pip?

**Comment by marscher** [comment]
fixed by version 0.2
